dotCMS/src/main/java/com/dotmarketing/portlets/templates/business/TemplateAPIImpl.java

```diff
@@ -6,17 +6,25 @@
 import com.dotcms.repackage.org.apache.oro.text.regex.MatchResult;
 import com.dotcms.repackage.org.apache.oro.text.regex.Perl5Compiler;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotcms.repackage.org.apache.oro.text.regex.Perl5Compiler'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)